kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/coordinatedcommits/CommitCoordinatorClientSuite.scala

```diff
@@ -144,6 +145,45 @@ class CommitCoordinatorClientSuite extends DeltaTableWriteSuiteBase {
     assert(cs1 ne cs1_again)
   }
 
+  test("Coordinated Commit Related Properties from Metadata") {
```

Comment:
just add a testsuite `TableConfigSuite` that tests the parsing logic instead of putting it here.
For `Engine`, the kernel-api module has a `mockEngine`